modules/ROOT/pages/cloudhub-api.adoc

```diff
@@ -126,6 +126,48 @@ Here is an example of data received in JSON format in response to a request to g
 }
 ----
 
+== Rate Limits 
+
+The CloudHub management API enforces usage limits.
+These limits don't apply to CloudHub workers.
+
```

Comment:
Should we add something here about "global" and perhaps imply the following two subtopics (exceptions)?

Reply:
Wow, this is vexatious. We could easily cover this in three bullets. Sigh.

Reply:
And these tables are butt-ugly.